lib/stores/FileStore.ts

```diff
@@ -49,19 +59,16 @@ class FileStore extends DataStore {
 
   /**
    * Create an empty file.
-   *
-   * @param  {File} file
-   * @return {Promise}
    */
-  create(file: any) {
+  create(file: File): Promise<File> {
     return new Promise((resolve, reject) => {
-      return fs.open(path.join(this.directory, file.id), 'w', async (err, fd) => {
+      fs.open(path.join(this.directory, file.id), 'w', (err, fd) => {
         if (err) {
           log('[FileStore] create: Error', err)
           return reject(err)
         }
 
-        await this.configstore.set(file.id, file)
```

Comment:
Why did you remove the await? Is the configstore synchronous?

Reply:
It never was async indeed

Reply:
Ok, but do we want it to be synchronous in the future as well? Maybe in the future we want to have a file-based config store, which needs to be async. Do you not think this could be the case?

Reply:
I personally prefer to not do premature optimizations. If we ever introduce it, it would be a non-breaking change to add an `await` and change the type signature.